maturity-level-two/README.md

```diff
@@ -11,3 +11,55 @@ You should:
 - Add your OpenAPI specs to source control given this is part of your application
 - Validate changes to your OpenAPI specs to avoid specification violations ([user guide](docs/validating-open-api-specs.md))
 - Unit test Open API validation to automatically detect breaking changes
+
+ ## Content negotiation
```

Comment:
Too long, this is looks more a documentation than a guideline.
Guideline should be a bullet list, this long description can be added as additional page in the docs folder and link from the readme

Reply:
@FilipVanRaemdonck let's sit together to bullet list the guideline